stackstorm-compose/README.md

```diff
@@ -0,0 +1,305 @@
+# StackStorm in Docker Compose
+
+This docker-compose is provided as a way to allow someone to "get up and running" quickly with StackStorm using Docker. It is not designed to be used in production, but rather a way to test out StackStorm and facilitate pack development.
+
+## TL;DR
+
+```shell
+git clone git@github.com:stackstorm/st2-dockerfiles
+cd st2-docker/stackstorm-compose
+docker-compose up -d
+docker-compose run st2api st2ctl reload --register-all  # you should only have to do this the first time
+docker-compose run st2client bash  # this gives you access to the st2 command line
```

Comment:
`exec` should be enough here, we don't need to spin up new container, do we?  

Also we I run this command, it always shows other components `##### st2 components status #####` as failed. I've searched but still don't know why. It should be just a wrong warning.

Reply:
RE: run vs exec.  If we do something like the `while true; do sleep 999; done` then we can switch from exec to run. As is, st2client will close out after each run.

Reply:
Right, we don't need a running container just to use a client. Let's think about this again after this one resolves: https://github.com/StackStorm/st2-dockerfiles/pull/28/files#r426154337

Reply:
However, I believe this command `docker-compose run st2api st2ctl reload --register-all` can be turned into exec, since st2api always runs in the background.

Reply:
> However, I believe this command `docker-compose run st2api st2ctl reload --register-all` can be turned into exec, since st2api always runs in the background.

Oh ok, I missed which run commnd you were referencing. Yes, this can be turned into an exec.

Reply:
st2client is now set to run all the time, and changed all the "docker-compose run" to "docker-compose exec".